sourcegraph/about
# Add Adam Berry podcast (season 3, episode 4)

### Changelog

- Added Adam Berry podcast (season 3, episode 4)

### Test

- Visit https://about.sourcegraph.com/podcast/adam-berry
  - Check YouTube video

- Visit https://about.sourcegraph.com/podcast/
  - Check MP3
